### PR TITLE
Math Magicians: tests

### PR DESCRIPTION
**Pull request by ragangithub**
- Used React Testing Library to simulate user interaction
- Used Jest
- Tested all components

### CODE REVIEW

File: src/test/Quote.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Quote from '../components/Quote';
+
+describe('Quote component', () => {
+  test('renders correctly', () => {
+    const component = render(<Quote />);
+    expect(component).toMatchSnapshot();
+  });
+});
```
Comment (ishimwezachee) [inline src/test/Quote.test.js:10]: - [ ] You should use jest snapshots for testing all the components from your project. It seemed like you've only tested Quote and Home component, while you have other components like Navbar, etc. 
Create the snapshots for the rest components as well to have well tested components of your project.

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @ragangithub  and @misalazeem , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Highlights
✅All linters are checked.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


## Required Changes ♻️

- [ ]  You did a great job in your project, but you've forgotten to add all contributors to this project in your project documentation/README.md fie . Please add all contributors to make this project more professional. 
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/Quote.test.js (hunk 1 above)
Comment (danmainah) [reply src/test/Quote.test.js:10]: - [ ] @ragangithub and @misalazeem great work on your tests, however as previously requested kindly add tests for the Navbar and Calculators components, its part of the project requirement.
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ragangithub and @misalazeem , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ Linter setup correctly and all checks are passing
✔️ Professional written README file with all sections required.


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @ragangithub @misalazeem , 
congratulations-1


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
All Tests Are Passing :heavy_check_mark: 
Made Requested Changes And Added Remaining Tests :heavy_check_mark: 
 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**